data/sql/updates/pending_db_world/rev_1678704026507325400.sql

```diff
@@ -0,0 +1,13 @@
+-- Ridgespine Horror
+UPDATE `creature_template_addon` SET `auras` = '7939 22766' WHERE `entry` = 20998;
+UPDATE `creature_template` SET `AIName` = 'SmartAI' WHERE `entry` = 20998;
+
+DELETE FROM `smart_scripts` WHERE `source_type` = 0 AND `entryorguid` = 20998;
+INSERT INTO `smart_scripts` (`entryorguid`, `source_type`, `id`, `link`, `event_type`, `event_phase_mask`, `event_chance`, `event_flags`, `event_param1`, `event_param2`, `event_param3`, `event_param4`, `event_param5`, `action_type`, `action_param1`, `action_param2`, `action_param3`, `action_param4`, `action_param5`, `action_param6`, `target_type`, `target_param1`, `target_param2`, `target_param3`, `target_param4`, `target_x`, `target_y`, `target_z`, `target_o`, `comment`) VALUES
+(20998, 0, 0, 0, 0, 0, 100, 0, 3000, 7000, 8000, 12000, 0, 11, 745, 0, 0, 0, 0, 0, 2, 0, 0, 0, 0, 0, 0, 0, 0, 'Ridgespine Horror - In Combat - Cast \'Web\''),
+(20998, 0, 1, 0, 0, 0, 100, 0, 0, 0, 5000, 12000, 0, 11, 7951, 0, 0, 0, 0, 0, 2, 0, 0, 0, 0, 0, 0, 0, 0, 'Ridgespine Horror - In Combat - Cast \'Toxic Spit\'');
+
+-- Ridgespine Stalker
+UPDATE `creature_template` SET `AIName` = '' WHERE `entry` = 20714;
+UPDATE `creature_template_addon` SET `auras` = '3616 7939 22766' WHERE `entry` = 20714;
```

Comment:
```
ServerToClient: SMSG_AURA_UPDATE (0x2C22) Length: 112 ConnIdx: 1 Time: 05/30/2022 19:05:58.246 Number: 200143
UpdateAll: True
AurasCount: 2
[0] Slot: 0
[0] HasAura: True
[0] CastID: Full: 0xBC44EC424007C0C30000330007107A9A Cast/3 R4411/S51 Map: 530 Low: 118520474
[0] SpellID: 7939 (7939)
[0] SpellXSpellVisualID: 0
[0] Flags: 0 (None)
[0] ActiveFlags: 0
[0] CastLevel: 66
[0] Applications: 0
[0] ContentTuningID: 0
[0] HasCastUnit: True
[0] HasDuration: False
[0] HasRemaining: False
[0] HasTimeMod: False
[0] PointsCount: 0
[0] EstimatedPoints: 0
[0] HasContentTuning: False
[0] CastUnit: Full: 0x2044EC4240143A8000003300000ECD59 Creature/0 R4411/S51 Map: 530 Entry: 20714 Low: 970073
[1] Slot: 1
[1] HasAura: True
[1] CastID: Full: 0xBC44EC4240163B830000330007907A9A Cast/3 R4411/S51 Map: 530 Low: 126909082
[1] SpellID: 22766 (22766)
[1] SpellXSpellVisualID: 248657
[1] Flags: 259 (NoCaster, Positive, MawPower)
[1] ActiveFlags: 3
[1] CastLevel: 66
[1] Applications: 0
[1] ContentTuningID: 0
[1] HasCastUnit: False
[1] HasDuration: False
[1] HasRemaining: False
[1] HasTimeMod: False
[1] PointsCount: 0
[1] EstimatedPoints: 0
[1] HasContentTuning: False
UnitGUID: Full: 0x2044EC4240143A8000003300000ECD59 Creature/0 R4411/S51 Map: 530 Entry: 20714 Low: 970073
```
These are correct but I don't see SpellID 3616

Reply:
https://www.youtube.com/watch?v=o29YE8nfqYM
https://www.youtube.com/watch?v=p5EQyl2T2jw
https://www.youtube.com/watch?v=-fLYSEjuYdw&
They sometimes get it off early/late, sometimes not at all with roughly the same combat time.

In mangos is with 3616 (proc aura), in TC is missing., here (used 744, the proc of 3616) was using on 20-80% hp SAI.
It's either 3616 via aura, or 744 as a SAI (unlikely due to the "rarity" in the videos)
https://www.wowhead.com/wotlk/npc=20714/ridgespine-stalker#abilities;mode:n25

Reply:
Blizzard isn't consistent. Different developers use different methods. With Thrash, for example (deals +2 attacks) it's sometimes triggered by aura, sometimes triggered by script, as it is easily sniffed.

In this case my experience has been 8-12s repeat, fairly consistent and skipping if aura is already applied (Flag 32, I think)

```
ServerToClient: SMSG_SPELL_START (0x2C3D) Length: 126 ConnIdx: 1 Time: 03/14/2023 20:42:25.443 Number: 3292
(Cast) CasterGUID: Full: 0x2041D84240143A800000380009908D69 Creature/0 R4214/S56 Map: 530 Entry: 20714 Low: 160468329
(Cast) CasterUnit: Full: 0x2041D84240143A800000380009908D69 Creature/0 R4214/S56 Map: 530 Entry: 20714 Low: 160468329
(Cast) CastID: Full: 0xBC41D8424000BA03000038003991067D Cast/3 R4214/S56 Map: 530 Low: 965805693
(Cast) OriginalCastID: Full: 0x0
(Cast) SpellID: 744 (744)
```

In a lot of cases procs don't have SPELL_START packets, this one has.

Reply:
Hmmm, interesting.